tests/shared/reducers/message.test.js

```diff
@@ -34,4 +37,34 @@ describe("reducers/app", () => {
       message: null,
     });
   });
+
+  describe("test addErrorToState function", () => {
```

Comment:
I would only put the name of the function here, e.g., `describe("addErrorToState()", ....`